prepare.php

```diff
@@ -15,6 +15,7 @@
 $WPT_SSH_OPTIONS = getenv( 'WPT_SSH_OPTIONS' ) ? : '-o StrictHostKeyChecking=no';
 $WPT_TEST_DIR = getenv( 'WPT_TEST_DIR' );
 $WPT_PHP_EXECUTABLE = getenv( 'WPT_PHP_EXECUTABLE') ? : 'php';
+$WPT_LOGGING = getenv( 'WPT_LOGGING') ? : 'normal';
```

Comment:
If we're not using the value "normal" for anything, would it make sense to leave it empty for the default value?